### PR TITLE
Promote thoughtbot's Elixir experience

### PR DESCRIPTION
**Pull request by derekprior**
We love working with Elixir and Phoenix and want to call attention
specifically to our experience with those technologies. Our
`elixir-phoenix` landing page has open source projects, blog posts, and
information on why we love Elixir.

### CODE REVIEW

**Comment by gfontenot** [comment]
👍

**Comment by paulcsmith** [comment]
I like this a lot. My concern is that this change is so far down the page that I'm not sure people will see it. I think it would be good to also have this near the top. Maybe something like this?

> ExMachina is part of [thoughtbot Elixir family].

I'm sure there is something better, but something along those lines might be good.

File: README.md [hunk 1]
```diff
@@ -284,11 +284,11 @@ redistributed under the terms specified in the [LICENSE](/LICENSE) file.
 ExMachina is maintained and funded by thoughtbot, inc.
 The names and logos for thoughtbot are trademarks of thoughtbot, inc.
 
-We love open source software!
-See [our other projects][community] or
-[hire us][hire] to design, develop, and grow your product.
+We love open source software, Elixir, and Phoenix.
+See [our other projects][elixir-phoenix],
+or [hire us][hire] to design, develop, and grow your product.
```
Comment (croaky) [inline README.md]: Possible to have some keywords in link text? eg "hire our Elixir Phoenix development team"

**Comment by derekprior** [comment]
@paulcsmith Updated. What do you think?

**Comment by paulcsmith** [comment]
@derekprior Yep that looks great! What do you think of moving it up and bolding it though?

screen shot 2016-06-17 at 11 49 32 am

I'm fine either way, but wanted to throw that out there as an option

**Comment by derekprior** [comment]
sure. will do.

**Pull request merged**